### ISSUE TITLE
CI failed due to UT tests failed(ZMQ related logs)

### ISSUE THREAD

**Issue by msosyak** [sonic-net/sonic-swss-common#784]
The CI is unstable and rarely passed the build stage due to timeout and the following logs
```
[ RUN      ] BinarySerializer.serialize_overflow
 ERROR:- setData: There are not enough buffer for binary serializer to serialize,                             key count: 1, data length 15, buffer size: 50
[       OK ] BinarySerializer.serialize_overflow (0 ms)
[ RUN      ] BinarySerializer.deserialize_overflow
 ERROR:- deserializeBuffer: serialized value data was truncated,, value length: 11 increase buffer size: 113


***

[ RUN      ] ZmqConsumerStateTable.test

***

# a lot of logs. No new test is started

 DEBUG:> select: enter
 DEBUG:< select: exit
 DEBUG:> select: enter
##[error]The operation was canceled.
```

For example, for PR #777  only 2 of 8  CI runs passed the build stage(Once before ZMQ was merged)

**Comment by msosyak** [comment]
@liuh-80 @qiluo-msft Could you have a look and help with that?

**Comment by msosyak** [comment]
Looks like fixed by #786
